eng/common/TestResources/New-TestResources.ps1

```diff
@@ -62,7 +62,10 @@ param (
     [switch] $CI = ($null -ne $env:SYSTEM_TEAMPROJECTID),
 
     [Parameter()]
-    [switch] $Force
+    [switch] $Force,
+
+    [Parameter()]
+    [switch] $File
```

Comment:
Add `[ValidateNotNullOrEmpty()]` if you want a value required without requiring the parameter.

Reply:
Do I want a value required?

Reply:
Oh, I just noticed this was a `[switch]`. I thought I saw `[string]` in which case, yes, you want a value. But since it's a switch, ignore that. However, the name is still confusing. `AsFile` would be a bit more consistent with built-in cmdlets, like `set-content -asbytestream`. Sorry.

Reply:
I'm not sure that I like AsFile, to me it reads a bit strange.

Reply:
`ToFile`? Just `File` is vague. This makes me feel like I need to specify a file path. Maybe `SaveFile`?

Reply:
OutFile seems pretty fine to me

Reply:
That works. Usually means a path, but in this case makes sense.